tests/safety/test_chrysler.py

```diff
@@ -96,8 +101,9 @@ def _speed_msg(self, speed):
     values = {"Vehicle_Speed": speed}
     return self.packer.make_can_msg_panda("ESP_8", 0, values)
 
+
 class TestChryslerRamHDSafety(TestChryslerSafety):
-  TX_MSGS = [[0x275, 0], [0x276, 0], [0x23A, 2]]
+  TX_MSGS = [[0x23A, 2], [0x275, 0], [0x276, 0]]
```

Comment:
this change makes the order consistent with the other platforms (button message first)